tests/cache/models/subjects/post.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/models/subjects/post.js
+ */
```

Comment:
I know we have lots of API tests under tests/cache/models/subjects but wondering if we want to move the api tests to its own folder, under tests/cache/api/subject/... going forward. ?